csharp-basic-topics/RemoveIllegalCharsFromPathAndFilenames/Tests/InvalidCharsTests.cs

```diff
@@ -0,0 +1,62 @@
+using CodeMazeGuides.CSBasic;
+using NUnit.Framework;
+
+namespace Tests
```

Comment:
You can use file scoped namespace declaration and get rid of the curly brackets block for the namespace